package.json

```diff
@@ -564,7 +564,7 @@
     },
     "scripts": {
         "vscode:prepublish": "yarn compile",
-        "compile": "tsc -p ./",
+        "compile": "yarn install && tsc -p ./",
         "watch": "tsc -watch -p ./",
         "pretest": "yarn compile",
         "test": "node ./out/src/test/runTest.js"
```

Comment:
I still think it would be good to have this do `yarn run pretest && node ...`.  Then update the ci.yml's to just call `yarn run test`

Reply:
Done.